specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-04-01-preview/examples/updateAKSCompute.json

```diff
@@ -0,0 +1,54 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "computeName": "compute123",
+    "api-version": "2020-03-01",
+    "parameters": {
+      "id": "compute123",
```

Comment:
ID is read-only, which cannot be in the request, please remove this.
This is reported by Model Validation.